test/compose-kafka-zookeeper.yaml

```diff
@@ -0,0 +1,18 @@
+version: "3"
+services:
+  zookeeper:
+    image: wurstmeister/zookeeper:latest
```

Comment:
would be better using strimzi images for this. Does them work fine right @scholzj? I remember that you suggested it to the Quarkus team for their tutorials.

Reply:
Yeah ... I would say so: https://github.com/scholzj/strimzi-compose-up